pushkin/request/event_handlers.py

```diff
@@ -87,16 +88,30 @@ def __init__(self, event_id, message_ids):
         self.message_ids = message_ids
 
     def handle_event(self, event, event_params):
+
+        # Allowed characters for parameter name are [a-zA-Z0-9_]
+        def get_parameter(param_name):
+            parameter = event_params.get(param_name)
```

Comment:
Code will break if parameter doesn't exists. Ideally we should log something in that case. Quick solution is to just return None or '' instead of param value.